color/src/colorspace.rs

```diff
@@ -540,22 +540,80 @@ impl ColorSpace for Rec2020 {
     }
 }
 
-/// 🌌 The ACEScg color space.
+/// 🌌 The ACES2065-1 color space.
 ///
-/// This color space is defined by the Academy Color Encoding System [specification][acescg].
+/// This is a linear color space with a very wide gamut. It is is often used for archival and
+/// interchange.
+///
+/// Its components are `[r, g, b]` (red, green, and blue channels respectively), with `[0, 0, 0]`
+/// pure black and `[1, 1, 1]` white. The natural bounds of the components are
+/// `[-65504.0, 65504.0]`.
+///
+/// This color space is [characterized by the Academy Color Encoding System][aces20651] and is
+/// specified in [SMPTE ST 2065-1:2021][smpte].
+///
+/// ACES2065-1 has a reference white [near D60][aceswp]; see the [XYZ-D65 color space](`XyzD65`)
+/// documentation for some background information on the meaning of "reference white."
+///
+/// See also [`AcesCg`].
+///
+/// [aces20651]: https://draftdocs.acescentral.com/specifications/encodings/aces2065-1/
+/// [smpte]: https://pub.smpte.org/doc/st2065-1/20200909-pub/st2065-1-2021.pdf
+/// [aceswp]: https://docs.acescentral.com/tb/white-point
+#[derive(Clone, Copy, Debug)]
+pub struct Aces2065_1;
```

Comment:
The underscore in the name is "correct," but also a bit awkward, as it's just a single digit trailing.